CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Considering https://repology.org/project/cmake/versions, maybe 3.10 is also justified?

Reply:
> Considering https://repology.org/project/cmake/versions, maybe 3.10 is also justified?

I'm not actually sure what specifically here requires 3.10, but would welcome any followup PR that changes the minimum version or adds a comment explaining the current minimum.

Reply:
> I'm not actually sure what specifically here requires 3.10

Nothing requires 3.10. Just thinking about availability of CMake 3.8 and 3.9 on real systems.